common/scripts/update-npm-dependencies.js

```diff
@@ -0,0 +1,126 @@
+const ncu = require('npm-check-updates');
+const Octokit = require('@octokit/rest');
+const { installAndRun, findRushJsonFolder, RUSH_JSON_FILENAME } = require("./install-run");
+const { readdirSync, existsSync } = require('fs');
+const { join } = require('path');
+const { spawnSync } = require('child_process');
+
+const { GITHUB_USERNAME, GITHUB_API_KEY } = process.env;
+const BRANCH_NAME = 'update-npm-dependencies';
+const COMMIT_MESSAGE = 'Update npm dependencies';
+const PR_TITLE = 'Update npm dependencies';
+const PR_BODY = `
+  This PR was auto-generated.
+  see https://github.com/binaris/shiftjs/blob/master/common/scripts/update-npm-dependencies.js
+`
+const IGNORED_PACKAGES = [
+  '@types/node'
+];
+
+function shellExec(command, args) {
+  const commandString = `${command} ${args.join(' ')}`;
+  console.log(commandString);
+  const { status } = spawnSync(command, args, {
+    stdio: 'inherit'
+  });
+
+  if (status !== 0) {
+    throw new Error(`'${commandString}' exited with code ${status}`);
+  }
+}
+
+async function updatePackageFiles() {
+  const allPackageFiles = readdirSync(process.cwd(), { withFileTypes: true })
```

Comment:
Come to think of it we should be able to get project directories out of rush.json - since rush repos often use depth of two for projects then this can be a general package that works with any rush repo.

Reply:
yes, this is what i meant by "separate" repo in a previous comment.
but then the `tsconfig` will not be shared ;)

Reply:
It can be in this repo and published to a package that is not shiftjs-...

Reply:
Moved everything to a new repo/package, so we need to merge https://github.com/binaris/rush-update/pull/1 before merging this

does look nicer now